testsuite/tests/letrec-compilation/lazy_.ml

```diff
@@ -1,5 +1,19 @@
 (* TEST *)
 
-let rec c = lazy (0 + d) and d = 3;;
+let rec c = lazy (0 + d) and d = 42;;
```

Comment:
Nitpick , but I don't think it is a good idea to use the same constant twice in a test (a bug might cause you to use the wrong value, but since you have used the same constant you don't detect it).